packages/java/engine-core/src/main/java/com/vaadin/hilla/engine/AotBrowserCallableFinder.java

```diff
@@ -102,19 +101,24 @@ private static Path generateAotArtifacts(
         var argsFile = engineConfiguration.getBuildDir()
                 .resolve("hilla-aot-args.txt");
         Files.write(argsFile, settings);
+        var report = engineConfiguration.getBuildDir()
+                .resolve("hilla-aot-report.txt");
```

Comment:
Nit: declare this variable as close as possible to where it is needed e.g. line 116.

Reply:
Made another change that makes that order irrelevant